js/intro.js

```diff
@@ -240,6 +256,15 @@ async function showSavedDetails() {
   try {
     const userId = urlParams.get('id');
     const usersRequest = await makeApiCall(`${BASE_URL}/users/${userId}/intro`);
+    const userInformation = await makeApiCall(
+      `${BASE_URL}/users/userId/${userId}`,
+    );
+
+    const dateDiff = getDateDifferenceInYearsAndMonths(
+      userInformation?.data.user.github_created_at,
```

Comment:
if `github_created_at` is not found then?

Reply:
This will return NaN but will not break UI as I have used optional chaining and have conditional rendering in frontend part

Reply:
OK, now understand, I will fix this

Reply:
done